docs/DotConfig.md

```diff
@@ -0,0 +1,48 @@
+Mock Firebolt: mf.config.json(Supported SDK's) 
```

Comment:
This file isn't quite what I had in mind, but it's okay for now. Once this PR gets merged (after changes based on this review), I can go in and make a few edits.

Reply:
okay